src/main/scala/org/mdedetrich/stripe/v1/Charges.scala

```diff
@@ -168,23 +169,31 @@ object Charges extends LazyLogging {
     }
   })
 
-  // Masked card
-
-  case class MaskedSource(
-      id: String,
-      last4: String,
-      expMonth: Int,
-      expYear: Int,
-      cvc: Option[String],
-      addressCountry: Option[String],
-      addressLine1: Option[String],
-      addressLine2: Option[String],
-      name: Option[String],
-      addressState: Option[String],
-      addressZip: Option[String]
-  ) extends MaskedCardSource
-
-  implicit val maskedSourceReads: Reads[MaskedSource] = (
+  sealed trait Source
```

Comment:
Can you make this a `sealed abstract class`? It just has better binary compatibility compared to `sealed trait`